relational_operators/CreateIndexOperator.hpp

```diff
@@ -55,19 +50,16 @@ class CreateIndexOperator : public RelationalOperator {
   /**
    * @brief Constructor.
    *
-   * TODO (ssaurabh): Modify the current placeholder implementation to
-   * take an index protobuf message as an input, instead of just name.
-   *
    * @param relation The relation to create index upon.
    * @param index_name The index to create.
-   * @param index_descriptions Set of index_descriptions associated with this index.
+   * @param index_description The index_description associated with this index.
    **/
   CreateIndexOperator(CatalogRelation *relation,
                       const std::string &index_name,
-                      const std::vector<IndexSubBlockDescription> &index_descriptions)
+                      IndexSubBlockDescription&& index_description)  // NOLINT
```

Comment:
@zuyu I tried using this syntax too, but somehow I am still getting the following lint error from style checker. 
`/catalog/CatalogRelation.hpp:270:  Missing spaces around &&  [whitespace/operators] [3]`
Is it something that I have change in the lint code itself to make this pass?